registry.json

```diff
@@ -4115,5 +4115,19 @@
    "telegram": "xbglowx",
    "operatorName": "Brian",
    "github": "xbglowx"
+  },
+  {
+   "ticker": "PFL01",
+   "poolId": "24bfb62716d755bbf035fcb3f7e87a8d325fde0f04062e692aff3216",
+   "social": {
+     "website": "https://poolforlovelace.me/",
+     "twitter": "PFL_pool",
+     "facebook": "",
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "PFL01",
```

Comment:
Please change this to be the telegram handle of an individual and not a group or channel